common/signal_processing/include/signal_processing/lowpass_filter.hpp

```diff
@@ -33,7 +33,7 @@ class LowpassFilterInterface
 public:
   explicit LowpassFilterInterface(const double gain) : gain_(gain) {}
 
-  void reset() { x_ = T{}; }
+  void reset() { x_ = {}; }
   void reset(const T & x) { x_ = x; }
```

Comment:
Changed the same way as
https://github.com/autowarefoundation/autoware.universe/blob/4c739d64cc0de0264c1122c26c97dd3646758c3e/common/signal_processing/src/lowpass_filter_1d.cpp#L19